nautobot-app/{{ cookiecutter.project_slug }}/tasks.py

```diff
@@ -688,26 +668,30 @@ def pylint(context):
 @task(aliases=("a",))
 def autoformat(context):
     """Run code autoformatting."""
-    black(context, autoformat=True)
     ruff(context, fix=True)
 
 
 @task(
     help={
-        "action": "One of 'lint', 'format', or 'both'",
-        "fix": "Automatically fix selected action. May not be able to fix all.",
-        "output_format": "see https://docs.astral.sh/ruff/settings/#output-format",
+        "action": "Available values are `['lint', 'format']`. Can be used multiple times. (default: `['lint']`)",
+        "fix": "Automatically fix selected actions. May not be able to fix all issues found. (default: False)",
+        "output_format": "See https://docs.astral.sh/ruff/settings/#output-format for details. (default: `full`)",
     },
+    iterable=["action"],
 )
-def ruff(context, action="lint", fix=False, output_format="text"):
+def ruff(context, action=["lint"], fix=False, output_format="full"):
```

Comment:
I'll still be on record as suggesting that the `lint`/`format` distinction in ruff is arbitrary and irrelevant to our needs. For example, applying isort rules is part of `lint` rather than part of `format`. Ruff is so fast, why not just run both lint and format every time?

Reply:
cc.: @Kircheneer

Reply:
Speed wouldn't be my concern. I don't feel too strongly, but not a fan of running a test (lint) and that conflicting with my not saved file and having a saved file conflict that I am confused as to how that happened. 

I will adjust my workflow and be more careful if we go this route, just providing the counter argument.

Reply:
Hmm, I can see both sides. WDYT of sticking with the current approach, just because that is where `ruff` itself draws the line? I can see that it is rather arbitrary, but there is an invoke command `inv ruff --action lint --action format` that runs both, so you don't really lose anything, right?

Reply:
I'm lazy, I like being able to just run `invoke ruff --fix` in core and have it automatically apply both linting and formatting fixes to my code. If for apps I have to run `invoke ruff --action lint --action format --fix` instead, I'll adjust, I'm just still not seeing the *benefit* of having it split into two separate commands.

Reply:
> Speed wouldn't be my concern. I don't feel too strongly, but not a fan of running a test (lint) and that conflicting with my not saved file and having a saved file conflict that I am confused as to how that happened.

Without specifying `--fix` to `invoke ruff`, it won't change your files with either action, so I'm not sure I understand the issue?

Reply:
Agreed, I'm lazy too, that's why there is `invoke a` (autoformat) :-).

There could be other tools than ruff fixing the code, that's why there is a separate action that should call all fixers.

The split makes sense to me as those `ruff check` and `ruff format` are different commands, some arguments (as `--output`) doesn't make to use with one of them.

Reply:
> > Speed wouldn't be my concern. I don't feel too strongly, but not a fan of running a test (lint) and that conflicting with my not saved file and having a saved file conflict that I am confused as to how that happened.
> 
> Without specifying `--fix` to `invoke ruff`, it won't change your files with either action, so I'm not sure I understand the issue?

I am responding to: `Ruff is so fast, why not just run both lint and format every time?``, I took every time as just that. 

I have no thoughts on the command structure (1 vs 2 commands), just responded to what was written.